rest-server/bin/harness-env

```diff
@@ -8,9 +8,9 @@ export REST_SERVER_HOST=${REST_SERVER_HOST:-localhost}
 export REST_SERVER_PORT=${REST_SERVER_PORT:-9090}
 
 # MongoDB setup
-# todo: allow for multiple servers in cluster
 export MONGO_HOST=${MONGO_HOST:-localhost}
 export MONGO_PORT=${MONGO_PORT:-27017}
+export MONGO_URI=${MONGO_URI:-mongodb://${MONGO_HOST}:${MONGO_PORT}}
```

Comment:
RE: https://github.com/actionml/harness/pull/171/files#r311305505
@pferrel this way?

Reply:
@andreyshikov needed but not enough

To be truly backward compatible we need to then derive `MONGO_HOST` and `MONGO_PORT` from `MONGO_URI` that way no matter what a user sets in the env, all 3 vars will be set correctly based on whatever is supplied OR defaults.

specifically any docker-compose or k8s usage will work. Then docker-compose or k8s can upgrade as they have time based on the new use of URIs.

By this logic we should do this with ALL service pointers, ES, etc.

Reply:
I think we should just merge this and worry about full backward compatibility only if needed. Our containers may break but we can fix them.